profile.md

```diff
@@ -672,32 +674,26 @@ In the following examples, a user has the following entitlements based on their
    </td>
   </tr>
   <tr>
-   <td>individual
+   <td><code>/microboone</code>
    </td>
-   <td><code>storage.read:/home/joe storage.write:/home/joe</code>
+   <td><code>storage.read:/microboone storage.write:/microboone/joe</code>
    </td>
   </tr>
   <tr>
    <td><code>/dune</code>
    </td>
-   <td><code>storage.read:/dune</code>
+   <td><code>storage.read:/dune storage.write:/dune/home/joe</code>
    </td>
   </tr>
   <tr>
    <td><code>/dune/dunepro</code>
    </td>
-   <td><code>storage.write:/dune/data</code>
-   </td>
-  </tr>
-  <tr>
-   <td><code>/microboone</code>
-   </td>
-   <td><code>storage.read:/microboone</code>
+   <td><code>storage.read:/dune storage.write:/dune/data</code>
    </td>
   </tr>
 </table>
 
-Since the user is a member of multiple groups (VOs) and also has the `/dune/dunepro` (production) membership, the resulting claims depend on the context indicated in the scope request:
+Since the user is a member of multiple groups (VOs) and also has the `/dune/dunepro` (production) membership, the resulting claims depend on the capability set indicated in the scope request:
```

Comment:
Could we rephrase the "has the `/dune/dunepro` (production) membership" into something like "is also member of the (production) group `/dune/dunepro`" ?

Reply:
Thanks for the suggestion.  I changed it to "is also a member of the `/dune/dunepro` (production) subgroup".